src/components/Category.js

```diff
@@ -74,9 +74,11 @@ const CategoryComponent = ({ category, stateContext }) => {
         })
 
         let updatedColumns = columns
+        .map(i => i.key === "Link" ? {...i, render: (link) => <a href={link} target="_blank" rel="noreferrer">{link}</a>} : i )
+        
         if (!isExternalResources) {
           updatedColumns = [
-            ...columns,
+            ...updatedColumns,
```

Comment:
The last time I checked this was causing issue due to re-render and double verification column was being added on all pages. Can you verify if this is not happening?

Reply:
ok

Reply:
Checked all tabs there isn't any duplication of verification columns.